src/compile/axis.ts

```diff
@@ -186,6 +186,14 @@ export function tickSize(model: Model, channel: Channel) {
   return undefined;
 }
 
+export function tickSizeEnd(model: Model, channel: Channel) {
```

Comment:
Do you mean we need to group them in to a) and b) that all the functions in a) are above b) and in alphabetical order?